lib/embeds.js

```diff
@@ -106,6 +106,14 @@ function renderCustom (props) {
   return '';
 }
 
+function renderTumblr (props) {
+  const { url, text } = props;
+  const href = text[0] && text[0].href ? text[0].href : url;
+
+  // There is no other way to show tumblr posts currently
+  return <a target='_blank' class='tumblr-post' href={href}>Load Tumblr post in new window</a>;
```

Comment:
I'd use `http://unwrapping.tumblr.com/post/101895859192/tumblr-sweatshirt-merch` instead, a bit nicer than generic text and (kinda) includes the post name.
@ellell what do you think?